tket/CMakeLists.txt

```diff
@@ -68,15 +73,15 @@ IF (CMAKE_CXX_COMPILER_ID STREQUAL "GNU")
 ENDIF()
 
 target_include_directories(tket PUBLIC include/tket)
-target_link_libraries(tket PRIVATE boost::boost)
-target_link_libraries(tket PRIVATE symengine::symengine)
-target_link_libraries(tket PRIVATE Eigen3::Eigen)
-target_link_libraries(tket PRIVATE nlohmann_json::nlohmann_json)
-target_link_libraries(tket PRIVATE tklog::tklog)
-target_link_libraries(tket PRIVATE tkassert::tkassert)
-target_link_libraries(tket PRIVATE tkrng::tkrng)
-target_link_libraries(tket PRIVATE tktokenswap::tktokenswap)
-target_link_libraries(tket PRIVATE tkwsm::tkwsm)
+target_link_libraries(tket PUBLIC boost::boost)
```

Comment:
Compiling the tests wasn't working with these as `PRIVATE`. If they need to be `PRIVATE` there might be a different way to get it working though...